samples/micrometer-samples-boot1/src/main/resources/application.yml

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-server.port: 8080
+server.port: 8083
```

Comment:
it doesn't look like this change belong to this PR

Reply:
Fixed